### DISCUSSION TITLE
Change visibility of some UI interfaces to public instead of internal

### DISCUSSION THREAD

**Discussion by xmnovotny** [voxeltycoon/issues#863] [Modding]
For better work with game UI (eg. adding new tabs to the existing windows), it would be nice to have interfaces like ```IWindowTabSelectHandler, IWindowTabDeselectHandler, IWindowTabFocusHandler, IWindowTabBlurHandler``` with public access, so all window tab functionality will be accessible for mods.

**Comment by andrewpey** [comment]
We'll do, thanks.

Made the issue for this: https://github.com/voxeltycoon/issues/issues/864.

**UPD** Done in `0.86.0.6` (available on the [beta branch](https://voxeltycoon.xyz/devlog/beta-branch/)).

**Reply by xmnovotny** [reply]
Thank you!
